docs/src/main/sphinx/security/apache-ranger-access-control.md

```diff
@@ -0,0 +1,168 @@
+# Apache Ranger access control
+
+The Apache Ranger access control plugin supports use of Apache Ranger policies to authorize data access in Trino on catalogs, schemas, tables, and columns. The plugin also supports column-masking, row-filtering and audit logging.
+
+## Requirements
+
+* Access to a Apache Ranger deployment with the desired authorization policies.
+* Access to an audit store using Solr, HDFS, Log4J, or S3 to save audit logs.
+* Apache Ranger 2.5.0 and greater include the required Trino service definition. Earlier versions of Apache Ranger require an update of the service definition available in the version [here](
+  https://github.com/apache/ranger/blob/ranger-2.5/agents-common/src/main/resources/service-defs/ranger-servicedef-trino.json).
+
+## Configuration
+
+To use only Ranger for access control, create the file `etc/access-control.properties` on the coordinator,
+with the following configuration, and configurations listed in the table below:
+
+```properties
+access-control.name=apache-ranger
+```
+
+
+To combine Ranger access control with file-based or other access control systems, create the file
+`etc/access-control.properties` on the coordinator, with the following configuration that lists
+multiple access control configuration file paths:
+
+```properties
+access-control.config-files=etc/trino/file-based.properties,etc/trino/apache-ranger.properties
+```
+
+Order the configuration files list in the desired order of the different systems
+for overall access control. Configure each access-control system in the
+specified files.
+
+The following table lists the configuration properties for the Ranger access control:
+
+:::{list-table} Apache Ranger access control configuration properties
+:widths: 25, 50, 25
+:header-rows: 1
+
+* - Name
+  - Description
+  - Default value
+* - `apache-ranger.service.name`
+  - Name of the service having policies to be enforced by the plugin
+  - dev_trino
+* - `apache-ranger.plugin.config.resource`
+  - List of Ranger plugin configuration files, comma separated. Relative paths will be resolved dynamically by searching in the classpath.
+  - ranger-trino-security.xml,ranger-trino-audit.xml,ranger-policymgr-ssl.xml
```

Comment:
Are defaults here and below are still accurate? I guess config now has empty lists of config files

Reply:
Removed default value column from this table, since there is no more default value for any of the listed configurations.